build/edgemark/hollow_edge_node_template.yaml

```diff
@@ -22,8 +22,6 @@ spec:
             - --name=$(NODE_NAME)
             - --http-server=https://{{server}}:10002
             - --websocket-server={{server}}:10000
-            - --alsologtostderr
-            - --v=2
```

Comment:
I think these two flag can use to debug, can we also explore how to enable them? Which is exised in other components :) @q77190858

Reply:
+1 

> remove these 2 args --alsologtostderr and --v=2, because edgemark now do not support , these args make edgemark exit error

maybe enable them instead.